desktop/core/src/desktop/lib/tasks/compress_files/compress_in_hdfs.sh

```diff
@@ -65,13 +65,22 @@ fi
 
 FILE_NAMES=("${FILE_NAMES//,/ }")
 exit_status=0
+failed=0
 
 temp_output_dir=`mktemp -d 2>/dev/null || mktemp -d -t 'mytmpdir'`
 echo 'Created temporary output directory: '$temp_output_dir
 
 set -x
-zip -r "$temp_output_dir/$ARCHIVE_NAME" "${FILE_NAMES[@]}"
```

Comment:
Wasn't the problem just because `"${FILE_NAMES[@]}"` currently expect a single file?

So just removing the " or quote each items? (maybe https://stackoverflow.com/questions/12985178/bash-quoted-array-expansion)

Reply:
you are correct, removing the quote works with multiple file without much of a code changes.  I will revert all the changes and remove just the quotes.